.circleci/config.yml

```diff
@@ -1002,39 +1005,51 @@ jobs:
             brew install cmake
       - with_hermes_tarball_cache_span:
           steps:
+            - persist_to_workspace:
```

Comment:
Persisting the `.hermes-cache-key-file` to the workspace to ensure that we use the same Hermes commit in all the jobs.